### ISSUE TITLE
Instance type not available

### ISSUE THREAD

**Issue by RainFlying** [alibaba/terraform-provider#466]
Got the following error:
```
* alicloud_instance.web.0: Instance type ecs.sn1ne.large is not supported in the availability zone cn-hongkong-b. Expected instance types of family ecs.sn1: ecs.sn1.xlarge(8Cores,16GB), ecs.sn1.3xlarge(16Cores,32GB), ecs.sn1.7xlarge(32Cores,64GB), ecs.sn1.large(4Cores,8GB), ecs.sn1.medium(2Cores,4GB).
```

According to the error message, "ecs.sn1ne.large" is not available in cn-hongkong-b, however, I created a few ecs.sn1ne.large instances in the same availability zone.
<img width="812" alt="screenshot 2018-05-07 17 27 44" src="https://user-images.githubusercontent.com/507678/39695096-fd8344f6-521b-11e8-80c4-a99539109c85.png">

**Comment by RainFlying** [comment]
After changed instance_type to "ecs.n4.large", got the follow error:

```
* alicloud_instance.web.0: Error creating Aliyun ecs instance: &common.Error{ErrorResponse:common.ErrorResponse{Response:common.Response{RequestId:"21C0B7E4-C0D6-4DE7-B3BB-90909746FB9C"}, HostId:"ecs-cn-hangzhou.aliyuncs.com", Code:"UnknownError", Message:"The request processing has failed due to some unknown error."}, StatusCode:400}
* alicloud_instance.web[1]: 1 error(s) occurred:
```

**Comment by RainFlying** [comment]
I'm using terraform-provider-V1.9.1

**Comment by ycsk02** [comment]
I met the same issue on V1.9.1, i confirmed with @xiaozhu36 this is a bug, he said would fix in next version, but this bug was not listed in V1.9.2 changelog.

I deployed with terraform in a supported instance type and changed it manually later.

**Comment by xiaozhu36** [comment]
Hi @RainFlying @ycsk02 I am sorry for late reply. This issue has been fixed by [PR](https://github.com/alibaba/terraform-provider/pull/472). Please be patient. Thanks a lot.

**Comment by RainFlying** [comment]
Fixed in version 1.9.3
